src/status_im2/contexts/quo_preview/transaction_sheet/transaction_sheet.cljs

```diff
@@ -0,0 +1,82 @@
+(ns status-im2.contexts.quo-preview.transaction-sheet.transaction-sheet
```

Comment:
this screen needs to be nuked- and replaced with a preview screen for locked-input